todo_app/mongo.py

```diff
@@ -0,0 +1,87 @@
+import os
+from bson.objectid import ObjectId
+import pymongo 
+from dotenv import load_dotenv
+from todo_app.todo_card import TodoCard
+from datetime import datetime
+
+class Mongo:
+    def __init__(self): 
+        self.user_name=os.getenv('USER_NAME')
+        self.password=os.getenv('PASSWORD')
+        self.mongo_url=os.getenv('MONGO_URL')
+        self.default_database=os.getenv('DEFAULT_DATABASE')
+        self.board_name=os.getenv('BOARD_NAME')
+        self.connection_string = "mongodb+srv://" + self.user_name + ":" + self.password + "@" + self.mongo_url + "/" + self.default_database + "?w=majority"
```

Comment:
No need to bother with a default_database env var when you specify the database name anyway, e.g. `client["To_Do_App"]`.

And rather than making board_name an env var, it might be nicer to make the database name an env var. So your tests definitely don't conflict with the real database.

Reply:
done

Reply:
done, added method to return the collection